openquakeplatform/openquakeplatform/static/js/irv_viewer.js

```diff
@@ -154,6 +155,17 @@ function combineIndicators(nameLookUp, themeObj, JSONthemes) {
             }
             subIndex[themeObjRegion] = tempElementValue;
         }
+    } else if (operator == 'Geometric mean (ignore weights)') {
+        for (var v5 = 0; v5 < themeObj.length; v5++) {
+            var tempElementValue = 1;
+            var themeObjRegion = themeObj[v5].region;
+            // compute the themes
+            for (var w5 = 0; w5 < themeKeys.length; w5++) {
+                var tempThemeName = themeKeys[w5];
+                tempElementValue = Math.sqrt(tempElementValue * (themeObj[v5][tempThemeName] * themeInversionFactor));
```

Comment:
This is correct only if you are combining 2 variables together. Otherwise, instead of elevating by 1/2, you need to elevate by 1/N, where N is the number of variables you are combining.